libbeat/scripts/Makefile

```diff
@@ -106,9 +106,11 @@ ${BEAT_NAME}.test: $(GOFILES_ALL)
 .PHONY: crosscompile
 crosscompile: ## @build Cross-compile beat for the OS'es specified in GOX_OS variable. The binaries are placed in the build/bin directory.
 crosscompile: $(GOFILES)
+ifneq ($(shell [[ $(BEAT_NAME) == journalbeat ]] && echo true ),true)
```

Comment:
Do we need this?  How about removing the thing that's invoking crosscompile for journalbeat (I assume this can just be dropped from the travis config).

Reply:
`crosscompile` was not added to Travis, so there was nothing to remove. I added the conditional here, so if someone tries to run it locally, it is skipped.